rollup.config.mjs

```diff
@@ -0,0 +1,27 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
+import sourcemaps from 'rollup-plugin-sourcemaps';
+
+export default {
+  input: './lib/index.js',
+  output: [{
+    file: './lib/bundle.mjs',
+    format: 'es',
+    preferConst: true,
+    sourcemap: true
+  }, {
+    file: './lib/bundle.cjs',
+    format: 'cjs',
+    preferConst: true,
+    sourcemap: true,
+    interop: false
+  }],
+  external: [
+    'clsx',
```

Comment:
Is `clsx` external?

Reply:
What this means is that `clsx` won't be included in the bundle, and it will be imported at runtime.
![image](https://user-images.githubusercontent.com/567105/96478050-1fba5480-122f-11eb-8226-b7d6efd520b7.png)
![image](https://user-images.githubusercontent.com/567105/96478072-2779f900-122f-11eb-997b-48eaec14ceba.png)